btf-gen/Dockerfile

```diff
@@ -34,10 +36,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Python install
 ENV PYENV_ROOT="/.pyenv"
 ENV PATH="$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH"
+ENV DEVA_NO_DYNAMIC_DEPS=$DEVA_NO_DYNAMIC_DEPS
 
 RUN git clone --depth=1 https://github.com/pyenv/pyenv.git ${PYENV_ROOT} && \
      pyenv install ${PYTHON_VERSION} && \
-     pyenv global ${PYTHON_VERSION}
+     pyenv global ${PYTHON_VERSION} && \
+     pip install "git+https://github.com/DataDog/datadog-agent-dev.git@${DEVA_VERSION}" && \
+     deva -v self dep sync -f legacy-build -f legacy-btf-gen && \
```

Comment:
Where is `legacy-btf-gen` defined?

Reply:
https://github.com/DataDog/datadog-agent-dev/blob/v0.4.3/pyproject.toml#L93-L99

Reply:
What is the motivation for storing the requirements out-of-repo?

Reply:
https://docs.google.com/document/d/1UqW697DrnSirSNycehB0PsOu5wrVg6PioCEdb8m33Hs/edit?pli=1&tab=t.0#heading=h.mf37kd2110de